pkg/new-ui/v1beta1/frontend/src/app/services/backend.service.ts

```diff
@@ -102,4 +102,49 @@ export class KWABackendService extends BackendService {
       .post(url, { postData: exp })
       .pipe(catchError(error => this.parseError(error)));
   }
+
+  getTrialLogs(name: string, namespace: string): Observable<any> {
+    const url = `/katib/fetch_trial_logs/?trialName=${name}&namespace=${namespace}`;
+
+    return this.http
+      .get(url)
+      .pipe(catchError(error => this.handleError(error, false)));
+  }
+
+  // ---------------------------Error Handling---------------------------------
+
+  // Override common service's getBackendErrorLog
+  // in order to properly show the message the backend has sent
+  public getBackendErrorLog(error: HttpErrorResponse): string {
+    if (error.error === null) {
+      return error.message;
+    } else {
+      // Show the message the backend has sent
+      return error.error.log ? error.error.log : error.error;
+    }
+  }
+
+  // Override common service's getErrorMessage
+  // in order to include the error.status in error message
+  public getErrorMessage(
+    error: HttpErrorResponse | ErrorEvent | string,
+  ): string {
+    if (typeof error === 'string') {
+      return error;
+    }
+
+    if (error instanceof HttpErrorResponse) {
+      if (this.getBackendErrorLog(error) !== undefined) {
+        return `[${error.status}] ${this.getBackendErrorLog(error)}`;
```

Comment:
IIUC this line is the only one that differs from the inherited function. I'd propose to not show the error in this case and not override this function, and just keep the changes from `getBackendError`.

Reply:
I removed the error status and now the message looks like:

![image](https://user-images.githubusercontent.com/87971102/218736252-01292807-023f-4d03-97e5-38b99b0ab4eb.png)